Update BIC highlights with note about otp_window (#257)
AdobeDocs/commerce-php: src/pages/development/backward-incompatible-changes/highlights.md

* [Magento_Quote](https://developer.adobe.com/commerce/php/module-reference/module-quote/)

### New system configuration validation for Two Factor Authentication `otp_window` value

> The `spomky-labs/otphp` library was updated which introduced a new validation requirement for supplying custom `otp_window` values. This configuration setting controls the grace period (in number of seconds) that is allowed for an expired one-time-password (OTP) to be used before it is denied. Previously it allowed for any number of seconds but now it must not be longer than the lifetime of a single OTP (usually 30 seconds). This means that the configuration value must now be changed if it is set longer.

Reviewer: ```suggestion
The `spomky-labs/otphp` library was updated which introduced a new validation requirement for supplying custom `otp_window` values. This configuration setting controls the grace period (in number of seconds) that is allowed for an expired one-time-password (OTP) to be used before it is denied. Previously, the library allowed any number of seconds to be specified, but now the value cannot be longer than the lifetime of a single OTP (usually 30 seconds).  This new requirement means that the configuration value must be changed if it is set longer.
```

> The `spomky-labs/otphp` library was updated which introduced a new validation requirement for supplying custom `otp_window` values. This configuration setting controls the grace period (in number of seconds) that is allowed for an expired one-time-password (OTP) to be used before it is denied. Previously, the library allowed any number of seconds to be specified. Now the value cannot be higher than the lifetime of a single OTP (usually 30 seconds).  This change means that the configuration value must be changed if it is set to 30 or higher.

Reviewer: ```suggestion
The `spomky-labs/otphp` library was updated which introduced a new validation requirement for supplying custom `otp_window` values. This configuration setting controls the grace period (in number of seconds) that is allowed for an expired one-time-password (OTP) to be used before it is denied. Previously, the library allowed any number of seconds to be specified. Now, the value cannot be higher than the lifetime of a single OTP (usually 30 seconds). This change means that the configuration value must be changed if it is set to 30 or higher.
```

> The `spomky-labs/otphp` library was updated which introduced a new validation requirement for supplying custom `otp_window` values. This configuration setting controls the grace period (in number of seconds) that is allowed for an expired one-time-password (OTP) to be used before it is denied. Previously, the library allowed any number of seconds to be specified. Now, the value cannot be higher than the lifetime of a single OTP (usually 30 seconds). This change means that the configuration value must be changed if it is set to 30 or higher.

Reviewer 2: ```suggestion
The updated `spomky-labs/otphp` library introduces a new validation requirement for supplying custom `otp_window` values. This configuration setting controls the grace period (in seconds) that an expired one-time-password (OTP) can be used until it is denied. Previously, the library allowed any number of seconds to be specified. Now, the value cannot be higher than the lifetime of a single OTP (usually 30 seconds). You must update this value if it is currently set to 30 or higher.

```

> The `spomky-labs/otphp` library was updated which introduced a new validation requirement for supplying custom `otp_window` values. This configuration setting controls the grace period (in number of seconds) that is allowed for an expired one-time-password (OTP) to be used before it is denied. Previously, the library allowed any number of seconds to be specified. Now, the value cannot be higher than the lifetime of a single OTP (usually 30 seconds). This change means that the configuration value must be changed if it is set to 30 or higher.

Reviewer 2: ```suggestion
The  updated`spomky-labs/otphp` library introduced a new validation requirement for supplying custom `otp_window` values. This configuration setting controls how long (in seconds) the system accepts an administrator's one-time-password (OTP) after it has expired. Previously, the library allowed any number of seconds to be specified. Now, the value cannot be higher than the lifetime of a single OTP (usually 30 seconds). If your configuration value is currently 30 or higher, you must change the value.
```

> If you are affected by this change, admin users may be unable to login and see the message `There was an internal error trying to verify your code`. You can confirm this by checking the `system.log` file in `var/log` for an entry `main.ERROR: The leeway must be lower than the TOTP period`.

Reviewer: ```suggestion
If your Commerce application is affected by this change, admin users might see the following message when they log in.  `There was an internal error trying to verify your code`. You can confirm the cause of the error by checking the `system.log` file in `var/log` for an entry `main.ERROR: The leeway must be lower than the TOTP period`.
```

> To fix this issue, you must change the value of the configuration path `twofactorauth/google/otp_window` to be shorter than the TOTP period which is usually 30 seconds. For example, you can reset it to the default value of `1` with the command `bin/magento twofactorauth/google/otp_window 1`. Note that you may need to flush the cache for this setting change to be enforced.

Reviewer: ```suggestion
To fix this issue, change the value of the configuration path `twofactorauth/google/otp_window` to be shorter than the TOTP period which is usually 30 seconds. For example, you can reset it to the default value of `1` with the command `bin/magento twofactorauth/google/otp_window 1`. You might need to flush the cache to apply the updated configuration. 
```

> To fix this issue, change the value of the configuration path `twofactorauth/google/otp_window` to be shorter than the TOTP period which is usually 30 seconds. For example, you can reset it to the default value of `1` with the command `bin/magento twofactorauth/google/otp_window 1`. You might need to flush the cache to apply the updated configuration.

Reviewer: ```suggestion
To fix this issue, change the value of the configuration path `twofactorauth/google/otp_window` to be shorter than the TOTP period which is usually 30 seconds. For example, you can reset it to the default value of `1` with the command `bin/magento config:set twofactorauth/google/otp_window 1`. You might need to flush the cache to apply the updated configuration.
```

> If your Commerce application is affected by this change, admin users might see the following message when they log in:  `There was an internal error trying to verify your code`. You can confirm the cause of the error by checking the `system.log` file in `var/log` for an entry `main.ERROR: The leeway must be lower than the TOTP period`.
> 
> To fix this issue, change the value of the configuration path `twofactorauth/google/otp_window` to be shorter than the TOTP period which is usually 30 seconds. For example, you can reset it to the default value of `1` with the command `bin/magento twofactorauth/google/otp_window 1`. You might need to flush the cache to apply the updated configuration. 

Reviewer: ```suggestion
If your Commerce application is affected by this change, admin users might see the following message when they log in: `There was an internal error trying to verify your code`. You can confirm the cause of the error by checking the `system.log` file in `var/log` for an entry `main.ERROR: The leeway must be lower than the TOTP period`.

To fix this issue, change the value of the configuration path `twofactorauth/google/otp_window` to be shorter than the TOTP period which is usually 30 seconds. For example, you can reset it to the default value of `1` with the command `bin/magento twofactorauth/google/otp_window 1`. You might need to flush the cache to apply the updated configuration.
```

> The `spomky-labs/otphp` library was updated which introduced a new validation requirement for supplying custom `otp_window` values. This configuration setting controls the grace period (in number of seconds) that is allowed for an expired one-time-password (OTP) to be used before it is denied. Previously, the library allowed any number of seconds to be specified. Now, the value cannot be higher than the lifetime of a single OTP (usually 30 seconds). This change means that the configuration value must be changed if it is set to 30 or higher.
> 
> If your Commerce application is affected by this change, admin users might see the following message when they log in: `There was an internal error trying to verify your code`. You can confirm the cause of the error by checking the `system.log` file in `var/log` for an entry `main.ERROR: The leeway must be lower than the TOTP period`.
> 
> To fix this issue, change the value of the configuration path `twofactorauth/google/otp_window` to be shorter than the TOTP period which is usually 30 seconds. For example, you can reset it to the default value of `1` with the command `bin/magento twofactorauth/google/otp_window 1`. You might need to flush the cache to apply the updated configuration.
> 

Reviewer: ```suggestion
The `spomky-labs/otphp` library was updated which introduced a new validation requirement for supplying custom `otp_window` values. Previously, the library allowed any number to be specified. Now, the value cannot be higher than the lifetime of a single OTP (usually 30 seconds).

If your Commerce application is affected by this change, admin users might see the following message when they log in: `There was an internal error trying to verify your code`. You can confirm the cause of the error by checking the `system.log` file in `var/log` for an entry `main.ERROR: The leeway must be lower than the TOTP period`.

To fix this issue, change the value of the configuration path `twofactorauth/google/otp_window`.   For example, you can reset it to the system default value of `1` with the command `bin/magento twofactorauth/google/otp_window 1`.  A value of `1` indicates that the current OTP code plus one code in the past and one code in the future remain valid at any given point in time.  Changing the value to `2` allows the OTP codes to remain valid for longer during automated testing.  You might need to flush the cache to apply the updated configuration.
```

> To fix this issue, change the value of the configuration path `twofactorauth/google/otp_window` to be shorter than the TOTP period which is usually 30 seconds. For example, you can reset it to the default value of `1` with the command `bin/magento config:set twofactorauth/google/otp_window 1`. You might need to flush the cache to apply the updated configuration.

Reviewer 2: ```suggestion
To fix this issue, change the value of the configuration path `twofactorauth/google/otp_window` to be shorter than the TOTP period, which is usually 30 seconds. For example, you can reset it to 29 seconds using the `bin/magento config:set twofactorauth/google/otp_window 29` command. You might need to flush the cache to apply the updated configuration.
```
